doc/user-guide/io.rst

```diff
@@ -815,24 +816,23 @@ Zarr Compressors and Filters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 There are many different `options for compression and filtering possible with
-zarr <https://zarr.readthedocs.io/en/stable/tutorial.html#compressors>`_.
+zarr <https://zarr.readthedocs.io/en/stable/user-guide/arrays.html#compressors>`_.
 
 These options can be passed to the ``to_zarr`` method as variable encoding.
 For example:
 
-.. ipython:: python
-    :suppress:
+.. jupyter-execute::
+    :hide-code:
 
     ! rm -rf foo.zarr
 
-.. ipython:: python
-    :okwarning:
+.. jupyter-execute::
 
     import zarr
-    from numcodecs.blosc import Blosc
+    from zarr.codecs import BloscCodec
 
-    compressor = Blosc(cname="zstd", clevel=3, shuffle=2)
-    ds.to_zarr("foo.zarr", encoding={"foo": {"compressor": compressor}})
+    compressor = BloscCodec(cname="zstd", clevel=3, shuffle="shuffle")
+    ds.to_zarr("foo.zarr", consolidated=False, encoding={"foo": {"compressors": [compressor]}})
```

Comment:
These changes apparently are necessary to work with zarr v3 (https://github.com/pydata/xarray/issues/10032)